src/scrape_up/finance/nse.py

```diff
@@ -90,5 +91,3 @@ def get_historical_data(self, from_date, to_date):
         return historical_price_data
 
 
-nse = NSE(stock_name="RELIANCE INDUSTRIES LTD")
-nse.get_latest_price()
```

Comment:
Still not working @brohithkr 
![image](https://github.com/Clueless-Community/scrape-up/assets/93156825/09cf7d41-c86a-474a-951f-33686e42f96c)

Reply:
Also the Docstring hasn't updatd.

Reply:
> ![image](https://user-images.githubusercontent.com/93156825/244474107-09cf7d41-c86a-474a-951f-33686e42f96c.png)

This is not working because reliance industries is a special case:

![Screenshot from 2023-06-09 06-31-19](https://github.com/Clueless-Community/scrape-up/assets/71490015/486ee081-66b3-4a48-ae07-f0bb71e7f3c3)

Its a company with something callled debt private placement. There is no priceInfo section for it.
In such cases you can just look up company's symbol from nse website(or google) and give it to nse class
Give "reliance" in this case.